Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml

```diff
@@ -31,6 +31,23 @@ configuration:
   defaultvalue: ""
   type: 0
   required: false
+- display: Fields
```

Comment:
let's add a description for both parameters

Reply:
don